ntoskrnl/se/sid.c

```diff
@@ -558,7 +608,9 @@ SeCaptureSidAndAttributesArray(
                                                  TAG_SID_AND_ATTRIBUTES);
         if (SidAndAttributes == NULL)
         {
-            return STATUS_INSUFFICIENT_RESOURCES;
+            DPRINT1("SeCaptureSidAndAttributesArray(): Failed to allocate memory for SID and attributes array!\n");
```

Comment:
Nit: printing the length of what failed to allocate is often helpful (i.e. was it a ridiculously large request vs are we out of memory)